### PR TITLE
WIP: e2e/framework: remove direct import to pkg/kubelet/apis/stats/v1alpha1

### PR DESCRIPTION
**Pull request by YuikoTakada**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
e2e/framework: remove direct import to pkg/kubelet/apis/stats/v1alpha1

**Which issue(s) this PR fixes**:

Ref #74352

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
@YuikoTakada thank you for this PR.

in my optioning duplicating the kubelet stats API in the framework is not something we should be doing. ideally this API should be extracted to a new repository that can be used by multiple consumers.

@kubernetes/sig-node-api-reviews 
@kubernetes/sig-node-pr-reviews 

we already have https://github.com/kubernetes/kubelet/tree/master/pkg/apis
so maybe that is the right place for it.

**Comment by YuikoTakada** [comment]
@neolit123 thank you for your comments. OK, I'll fix it. I checked https://github.com/kubernetes/kubelet/tree/master/pkg/apis but I don't know to where should I move some struct and const. what do you think?

**Comment by neolit123** [comment]
@YuikoTakada hi, it is not clear if kubernetes/kubelet is the right repository for this.
if this PR is not getting a response you can try asking in the #sig-node channel or in their zoom meeting.

**Comment by YuikoTakada** [comment]
@neolit123 thank you. OK, I'll consider for a while.

File: test/e2e/framework/kubelet/stats.go [hunk 1]
```diff
@@ -53,6 +52,13 @@ const (
 	// dockerOperationsTimeoutKey is the key for the operation timeout metrics.
 	// copied from k8s.io/kubernetes/pkg/kubelet/dockershim/metrics
 	dockerOperationsTimeoutKey = "docker_operations_timeout_total"
+
+	// SystemContainerRuntime is the container name for the system container tracking the runtime (e.g. docker) usage.
+        // NOTE: Copied from pkg/kubelet/apis/stats/v1alpha1 to avoid pulling extra dependencies
```
Comment (oomichi) [inline test/e2e/framework/kubelet/stats.go:57]: indents seems odd against the other previous lines.

**Review by oomichi — COMMENTED** [review]
/cc @oomichi

**Pull request closed without merging**